### PR TITLE
v5.0.0-beta.4 – Updating colours to use new pie-design-tokens variable equivalents

### PR DESCRIPTION
**Pull request by ashleynolan**
### Changed
- Moved from using `fozzie-colour-palette` to `pie-design-tokens`.
- Colour variables transitioned:
  - `$black` > `$color-black`
  - `$white` > `$color-white`
  - `$grey--light` > `$color-grey-30`
  - `$brand--orange` > `$color-orange-30`
  - `$orange` > `$color-orange`
  - `$orange-dark` > `$color-orange` & darken(4%)
  - `$orange-darkest` > `$color-orange` & darken(10%)
  - `$orange--aa` > `$color-orange-60`
  - `$orange--offWhite` > `$color-orange-10`
  - `$blue` > `$color-blue`
  - `$blue--offWhite` > `$color-blue-10`
  - `$blue--offWhite--dark`> `$color-blue-10` & darken(4%)
  - `$blue--offWhite--darkest`> `$color-blue-10` & darken(10%)
  - `$red` > `$color-red`
  - `$green` > `$color-green`
  - `$green--offWhite` > `$color-green-10`
  - `$yellow--offWhite` > `$color-yellow-10`
  - `$grey--offWhite` > `$color-grey-10`
  - `$grey--lighter` > `$color-grey-20`
  - `$grey--mid` > `$color-grey-40`
  - `$grey--dark` > `$color-grey-50`
  - `$grey--darkest` > `$color-grey`
  - `$purple` > `$color-purple`
  - `$purple--light` > `color-purple-10`

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines

### CODE REVIEW

File: src/scss/base/_typography.scss [hunk 1]
```diff
@@ -226,8 +226,7 @@
     hr {
         margin: ($baseline) 0;
         border: 0;
-        border-top: 1px solid $hr-color;
-        border-bottom: 1px solid $white;
```
Comment (ashleynolan) [inline src/scss/base/_typography.scss:230]: This was part of the old HR styling, that is no longer used.

**Review by zlatinivanov-je — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

File: src/scss/components/_cards.scss [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
     $card-tooltip-width                             : 10px;
     $card-arrow-bottom-position                     : 0;
     $card-padding                                   : spacing(x2);
     $card-radius                                    : 4px;
     $card-borderColor                               : $color-border;
     $card-section-margin                            : spacing(x4);
     $card-section-margin--large                     : spacing(x8);
-    $card-section-highlight-backgroundColor         : $orange--offWhite;
+    $card-section-highlight-backgroundColor         : $color-orange-10;
     $card-section-highlight-color                   : $color-text;
```
Comment (DamianMullins) [inline src/scss/components/_cards.scss:24]: I was just looking at upgrading to the latest beta version on menu and I see errors around the `$color-text` variable, it looks like it no longer exists in PIE, is that correct? And if so what should it be replace with?

Comment (dandel10n) [reply src/scss/components/_cards.scss:24]: @DamianMullins I checked that it should be `$color-content-default` now.

Comment (DamianMullins) [reply src/scss/components/_cards.scss:24]: Thanks @dandel10n, I'll update this as it was causing issues for me.

Comment (DamianMullins) [reply src/scss/components/_cards.scss:24]: OK, it looks like there are other variables that also to be updated, I'll dig around for any tickets around this work. It feels like something that Ash probably already had in progress.

**Pull request merged**